packages/api/src/command/medical/document/document-query.ts

```diff
@@ -64,8 +64,7 @@ export async function queryDocumentsAcrossHIEs({
     reset: true,
   });
 
-  // intentionally asynchronous, not waiting for the result
-  getDocumentsFromCW({ patient, facilityId, override });
+  getDocumentsFromCW({ patient, facilityId, override }).catch(emptyFunction);
```

Comment:
Empty handler logic to avoid log when an issue is thrown on `getDocumentsFromCW` (it already does error handling there).